logic/src/commonMain/kotlin/com/wire/kalium/logic/data/conversation/ConversationMapper.kt

```diff
@@ -91,8 +92,17 @@ internal class ConversationMapperImpl(
         }
     }
 
+    override fun toDAOGroupState(groupState: Conversation.ProtocolInfo.MLS.GroupState): GroupState =
```

Comment:
since all the states are the same, isn't good idea to map them with ordinal number in one line -> like
Conversation.Protocol.Info.MLS.values()[groupstate.ordinal] ...

Reply:
Seems fragile since then we rely on that the ordering is the same.